### PR TITLE
add frequency values validator

### PR DESCRIPTION
**Pull request by delorge**

### CODE REVIEW

File: src/validators/isFrequency.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import {unit} from 'postcss-value-parser';
+import endsWith from 'ends-with';
+
+const frequences = [
```
Comment (ben-eb) [inline src/validators/isFrequency.js]: The proper plural is `frequencies`. 😉

File: src/validators/isFrequency.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'Hz',
+    'kHz',
+];
+
+export default ({type, value}) => {
+    console.log(type, value);
```
Comment (ben-eb) [inline src/validators/isFrequency.js]: This can be removed! 😄

**Comment by delorge** [comment]
whoops, forgot to remove :D

**Comment by ben-eb** [comment]
Awesome, thanks! 👍

**Pull request merged**